### PR TITLE
Update business rules documentation

### PR DESCRIPTION
**Pull request by mohierf**

### CODE REVIEW

File: source/06_alignak_features/business_rules.rst [hunk 1]
```diff
@@ -0,0 +1,608 @@
...
+
+This makes it compatible with all your current habits and UIs. As the service aggregation is
+considered as any other state from a host or service, you can get notifications, actions and
+escalations. This means you can have contacts that will receive only the relevant
+notifications based on their role.
+
+.. warning::  You must NOT define the `bp_rule` command, it's purely internal and Alginak already defined it for you.
+
+
+Here is a configuration example for the ERP service example, attached to a dummy host named "servicedelivery".
```
Comment (ddurieux) [inline source/06_alignak_features/business_rules.rst]: `example` + `example` = too `example` :p

File: source/06_alignak_features/business_rules.rst [hunk 2]
```diff
@@ -0,0 +1,608 @@
...
+
+The aggregated state is somehow like a boolean property:
+
+    * db1 OK | db2 OK -> Simple_Or will be OK
+    * db1 OK | db2 WARNING -> Simple_Or will be OK
+    * db1 OK | db2 CRITICAL -> Simple_Or will be OK
+    * db1 WARNING | db2 OK -> Simple_Or will be OK
+    * db1 WARNING | db2 WARNING -> Simple_Or will be WARNING
+    * db1 CRITICAL | db2 OK -> Simple_Or will be OK
+    * db1 CRITICAL | db2 WARNING -> Simple_Or will be OK
```
Comment (ddurieux) [inline source/06_alignak_features/business_rules.rst]: Will not be OK :p

File: source/06_alignak_features/business_rules.rst [hunk 3]
```diff
@@ -0,0 +1,608 @@
...
+    * db1 CRITICAL & db2 OK -> Simple_And will be CRITICAL
+    * db1 CRITICAL & db2 WARNING -> Simple_And will be CRITICAL
+    * db1 CRITICAL & db2 CRITICAL -> Simple_And will be CRITICAL
+
+
+The NOT rule
+============
+
+
+You can define a not state rule. It can be useful for active/passive setups for example.
```
Comment (ddurieux) [inline source/06_alignak_features/business_rules.rst]: Typo:
You can define a `NOT` state rule.

File: source/06_alignak_features/business_rules.rst [hunk 4]
```diff
@@ -0,0 +1,608 @@
...
+
+
+
+With "need at least X elements" clusters
+========================================
+
+Sometimes, you know that in a cluster of N elements, you need at least X of them to run OK. This
+is easily defined, you just need to use the `X of:` operator.
+
+Here is an example of the same ERP but with 3 http web servers, and you need at least 2 of them
```
Comment (ddurieux) [inline source/06_alignak_features/business_rules.rst]: typo: `http` => `HTTP`

File: source/06_alignak_features/business_rules.rst [hunk 5]
```diff
@@ -0,0 +1,608 @@
...
+(to maintain the load):
+
+
+::
+
+  define service{
+         use                  standard-service
+         host_name            servicedelivery
+         service_description  ERP
+         check_command        bp_rule!(h1,database1 | h2,database2) & (2 of: h3,Http1 & h4,Http4 & h5,Http5) & (h6,IPVS1 | h7,IPVS2)
```
Comment (ddurieux) [inline source/06_alignak_features/business_rules.rst]: Remove ` & (h6,IPVS1 | h7,IPVS2)` to be more clear ;)

File: source/06_alignak_features/business_rules.rst [hunk 6]
```diff
@@ -0,0 +1,608 @@
...
+
+
+
+Manage degraded status
+=======================
+
+
+In the ``Xof:`` way the only case where you got a "warning" (="degraded but not dead")
+is when all your elements are in WARNING state. But you should want to be in WARNING if 1 or your
+3 http server is CRITICAL: the service is still running, but in a degraded state.
```
Comment (ddurieux) [inline source/06_alignak_features/business_rules.rst]: typo: `http` => `HTTP`

File: source/06_alignak_features/business_rules.rst [hunk 7]
```diff
@@ -0,0 +1,608 @@
...
+=============================
+
+It's a simple service (or an host) with a "special" check_command named ``bp_rule``. :)
+
+This makes it compatible with all your current habits and UIs. As the service aggregation is
+considered as any other state from a host or service, you can get notifications, actions and
+escalations. This means you can have contacts that will receive only the relevant
+notifications based on their role.
+
+.. warning::  You must NOT define the `bp_rule` command, it's purely internal and Alginak already defined it for you.
```
Comment (mohierf) [inline source/06_alignak_features/business_rules.rst]: Ok

File: source/06_alignak_features/business_rules.rst (hunk 1 above)
Comment (mohierf) [inline source/06_alignak_features/business_rules.rst]: Ok

File: source/06_alignak_features/business_rules.rst (hunk 2 above)
Comment (mohierf) [inline source/06_alignak_features/business_rules.rst]: Should be WARNING ... bad copy/paste :/

**Pull request merged**